### PR TITLE
Create Rahul-Prasad-07.json

### PR DESCRIPTION
**Pull request by Rahul-Prasad-07**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
kindly review my pr and merge it

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2567"></a>

### CODE REVIEW

File: data/Rahul-Prasad-07.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "Rahul Prasad",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " Blockchain Developer | **Open Source** Advocate | DevOps Developer",
+  "tags": ["Open Source","Blockchain", "DevOps", "Kubernetes", "Docker", "Node.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/rahultwte" },
+    { "icon": "FaGithub", "url": "https://github.com/Rahul-Prasad-07" },
+    { "icon": "website", "url": "https://dev.to/rahulprasad07" }
```
Comment (eddiejaoude) [inline data/Rahul-Prasad-07.json:10]: This should be an icon from the icon sets we approve, you can search for them here https://linkfree.eddiehub.io/icons

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
1 icon needs updating

File: data/Rahul-Prasad-07.json (hunk 1 above)
Comment (srkds) [inline data/Rahul-Prasad-07.json:10]:
```suggestion
    { "icon": "FaLink", "url": "https://dev.to/rahulprasad07" }
```

Comment (eddiejaoude) [reply data/Rahul-Prasad-07.json:10]: I think this is a good icon to use 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**